scripts/load-gotland.sh

```diff
@@ -376,6 +376,24 @@ while [ "$year" -le 2019 ]; do
 	year=$(( year + 1 ))
 done | psql --quiet
 
+# Fix up the body fat data
+cat <<-END_SQL
```

Comment:
```suggestion
psql --quiet<<-'END_SQL'
```
Then it loads correctly

Reply:
Yes, that's the correct thing to do.